web/modules/custom/collection_page/config/install/field.field.node.collection_page.og_group_ref.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
```

Comment:
Nice to see that this is an OG content type. Is this actually working? Do you think it's worthwhile to test this at this point? Or is there perhaps a future ticket that plans a full UAT test for this?